src/CoreL2/MacController/MacController.cpp

```diff
@@ -0,0 +1,228 @@
+#include "MacController.h"
+
```

Comment:
Coloque um mínimo de comentário sobre a funcionalidade da classe, como fez em alguns casos...

Reply:
Você diz o header padrão? Estranho, eu verifiquei e todos os arquivos .cpp têm o header padrão no modelo:


`/* ***************************************/
/* Copyright Notice                      */
/* Copyright(c)2020 5G Range Consortium  */
/* All rights Reserved                   */
/*****************************************/
/**
@Arquive name : MacController.cpp
@Classification : MAC Controller
@
@Last alteration : January 20th, 2020
@Responsible : Eduardo Melao
@Email : emelao@cpqd.com.br
@Telephone extension : 7015
@Version : v1.0

Project : H2020 5G-Range

Company : Centro de Pesquisa e Desenvolvimento em Telecomunicacoes (CPQD)
Direction : Diretoria de Operações (DO)
UA : 1230 - Centro de Competencia - Sistemas Embarcados

@Description : This module manages all System execution modes and their transitions. It creates and starts TUN Interface reading, Control SDUs creation, Sending timeout control, and decoding threads. Also, management of SDUs concatenation into PDU is made by this module.
*/`

Reply:
Pode ser que essas alterações sejam de um commit antigo